CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 
 - Fix checkbox/switch validity status
+- Renamed Card component's `outlined` property to `elevated`.
+- Renamed Calendar component's `hasHeader` property & `has-header` attribute to `hideHeader` & `hide-header` respectively.
```

Comment:
Since the properties are not just renamed but their logic is inverted I would suggest:
```suggestion
- Replaced Card component's `outlined` property with `elevated`.
- Replaced Calendar component's `hasHeader` property & `has-header` attribute with `hideHeader` & `hide-header` respectively.
```

Reply:
Also maybe we need to highlight it as a breaking change.

Reply:
Thinking about this further we may want to first deprecate the old props as [Semantic Versioning](https://semver.org/spec/v2.0.0.html#how-should-i-handle-deprecating-functionality) suggests.